recipes/mender-monitor/debian-master/control

```diff
@@ -0,0 +1,15 @@
+Source: mender-monitor
+Maintainer: Mender Team <mender@northern.tech>
+Section: admin
+Priority: optional
+Standards-Version: 3.9.2
+Build-Depends: debhelper (>= 9)
+
+Package: mender-monitor
+Architecture: all
+Depends: mender-client (>=3.0) ${misc:Depends}
```

Comment:
It depends on mender-client 2.5 (requires only auth D-Bus API) and `lmdb` (also bash? :thinking: ) See [dependencies from yocto](https://github.com/mendersoftware/meta-mender/blob/master/meta-mender-commercial/recipes-mender/mender-monitor/mender-monitor.inc#L6).